From c50889a46b58a46b8e64028328f68626269b7e4a Mon Sep 17 00:00:00 2001
From: Daz Jones <yuki@thebronasium.com>
Date: Fri, 7 Feb 2014 01:41:29 +0100
Subject: [PATCH] minui: display-legacy says NO

ergh
---
 minui/Android.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/minui/Android.mk b/minui/Android.mk
index 2d3198a..640bb15 100644
--- a/minui/Android.mk
+++ b/minui/Android.mk
@@ -17,10 +17,6 @@ ifeq ($(call is-vendor-board-platform,QCOM),true)
   LOCAL_C_INCLUDES += $(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include
 endif
 
-ifeq ($(TARGET_USES_QCOM_BSP), true)
-    LOCAL_CFLAGS += -DMSM_BSP
-endif
-
 LOCAL_MODULE := libminui
 
 # This used to compare against values in double-quotes (which are just
-- 
1.8.1.2

